piqe_utils/api/vm_ops/vm_provider/libvirt/libvirtd.py

```diff
@@ -0,0 +1,121 @@
+"""Module to perform libvirtd related operations"""
+import sys
+from paramiko import SSHClient
+import os
+
+# getting the name of the directory
+# where the this file is present.
+current_dir = os.path.dirname(os.path.realpath(__file__))
+# the sys.path.
+sys.path.append(f"{current_dir}/../../../../../")
+from piqe_utils.api.utils import get_output_local, get_output_remote
+
+service_name = "libvirtd.service"
```

Comment:
FYI: https://libvirt.org/daemons.html#modular-driver-daemons
The service is extending to several different daemon, so I guess we can define a list for service_name: `service_name = ["libvirtd.service"]`, and extend the list in the future when needed.

Reply:
gotcha, thanks